specification/vision/ImageAnalysis/models.tsp

```diff
@@ -0,0 +1,324 @@
+import "@typespec/http";
+
+namespace ImageAnalysis;
+using TypeSpec.Http;
+
+@doc("Query parameters used for the analyze operator")
+model SharedAnalyzeQuery {
+  @query({
+    name: "features",
+    format: "csv",
+  })
+  @doc("A string indicating what visual feature types to return. Multiple values should be comma-separated. Valid visual feature types include: Tags, Caption, DenseCaptions, Objects, Read, SmartCrops, People. At least one visual feature must be specified for Image Analysis.")
+  visualFeatures?: Array<VisualFeatures>;
+
+  @query("language")
+  @doc("The desired language for output generation. If this parameter is not specified, the default value is \"en\". See https://aka.ms/cv-languages for a list of supported languages.")
+  language?: string = "en";
+
+  @query("gender-neutral-caption")
+  @doc("Boolean flag for enabling gender-neutral captioning for caption and denseCaptions features. If this parameter is not specified, the default value is \"false\".")
+  genderNeutralCaption?: boolean = false;
+
+  @query({
+    name: "smartcrops-aspect-ratios",
+    format: "csv",
+  })
+  @doc("A list of aspect ratios to use for smartCrops feature. Aspect ratios are calculated by dividing the target crop width by the height. Supported values are between 0.75 and 1.8 (inclusive). Multiple values should be comma-separated. If this parameter is not specified, the service will return one crop suggestion with an aspect ratio it sees fit between 0.5 and 2.0 (inclusive).")
+  smartCropsAspectRatios?: Array<float32>;
+
+  @query("model-name")
+  @doc("The name of the custom trained model. This parameter needs to be specified if the parameter \"features\" is not specified.")
+  modelName?: string;
+}
```

Comment:
nit, it usually preferred to order the parameters for mostly important ones at top. (e.g. it may help if having model-name before language)

Also it may help to refer to current Swagger https://github.com/Azure/azure-rest-api-specs-pr/blob/290cc7baf21d292453e57aaa0bc014a7e3e4e01c/specification/cognitiveservices/data-plane/UnifiedVision/preview/2023-04-01-preview/UnifiedVision.json#L637-L662

Reply:
@weidongxu-microsoft thanks for your comment! yes, I did try to order them based on the most-used based on the telemetry reports of the current SDK in public preview. 'model-name' is only needed when you do Image Analysis with a custom-trained AI-model. That is an advanced scenario, with little usage compared to Image Analysis with a standard AI model, hence I put it last.